src/main/java/org/fundacionjala/fc/config/PropertiesReader.java

```diff
@@ -0,0 +1,90 @@
+package org.fundacionjala.fc.config;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+
+/**
+ * PropertiesReader class.
+ */
+public final class PropertiesReader {
+
+    private FileReader reader;
+    private Properties property;
+    private static final String PATH = "gradle.properties";
+    private static PropertiesReader singleInstance;
+
+    private PropertiesReader() {
+        try {
+            reader = new FileReader(PATH);
+            property = new Properties();
+            property.load(reader);
+
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
```

Comment:
Please consider using a logger instead of `printStackTrace()`.
```java
// First declare a logger class variable:
private static final Logger LOGGER = LogManager.getLogger(Environment.class);

// Second add logger statements to catch definitions and throw InitializationError class
} catch (FileNotFoundException e) {
    LOGGER.error("Error when reading file");
    LOGGER.error(e.getMessage());
    // Thrown error here
}
```